src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -132,17 +138,38 @@ private static Properties loadProperties() {
     return properties;
   }
 
-  private static String getUserAgent() {
+  /**
+   * Creates a string for user agent which should always be present in all requests to Snowflake
+   * (Snowpipe APIs) Here is the format we will use SnowpipeJavaSDK/version (platform details)
+   * JAVA/<java-version>
+   *
+   * @return the default agent string
+   */
+  private static String getDefaultUserAgent() {
```

Comment:
@sfc-gh-stakeda if you can just review this part of the code.

Reply:
This method looks ok to me.